manifests/deploy/openshift/olm/bundle/4.1/reportdatasource.crd.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: reportdatasources.metering.openshift.io
+spec:
+  group: metering.openshift.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: reportdatasources
+    singular: reportdatasource
+    kind: ReportDataSource
+    shortNames:
+    - datasource
+    - datasources
+  additionalPrinterColumns:
+  - name: Earliest Metric
+    type: string
+    JSONPath: .status.prometheusMetricImportStatus.earliestImportedMetricTime
```

Comment:
I believe this needs to be updated now that #706 was just merged.

Reply:
Ah, I should have done that in the other PR. Fixed here.